packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts

```diff
@@ -83,6 +84,8 @@ export const BRANDING_DEFAULT: BrandingData = {
       'https://www.eclipse.org/che/docs/stable/end-user-guide/url-parameter-for-the-workspace-storage/',
     webSocketTroubleshooting:
       'https://www.eclipse.org/che/docs/stable/end-user-guide/troubleshooting-network-problems/',
+    startingAWorkspaceFromAGitRepositoryURL:
```

Comment:
Few remarks:
1) Variable name could be simplified and not linked to docs article name, which could be changed in future; e.g. `startWorkspaceFromGit`
2) Don't forget to add downstream documentation URL for `startingAWorkspaceFromAGitRepositoryURL` into the **product.json**: https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-dashboard/packages/dashboard-frontend/assets/branding/product.json#L21

Reply:
Done, pull request for the downstream: https://github.com/redhat-developer/devspaces-images/pull/658